amazon_transcribe/model.py

```diff
@@ -224,6 +225,8 @@ class StartStreamTranscriptionRequest:
         results. A higher stability level means that the transcription results
         are less likely to change. Higher stability levels can come with lower
         overall transcription accuracy.
+    :param language_model_name
+        The name of the pre-existing language model to be used during transcription.
```

Comment:
```suggestion
        The name of the language model you want to use.
```

Reply:
@nateprewitt this is different text than requested at line 317 in model.py (https://github.com/awslabs/amazon-transcribe-streaming-sdk/pull/70#discussion_r843333616)

Is this intentional?

Reply:
That's correct. This is the `StartStreamTranscriptionRequest` object and  https://github.com/awslabs/amazon-transcribe-streaming-sdk/pull/70#discussion_r843333616 is the `StartStreamTranscriptionResponse` object. They have different wording based on the direction the action.

You can find the official docs for each [here](https://docs.aws.amazon.com/transcribe/latest/APIReference/API_streaming_StartStreamTranscription.html).